scene/3d/voxel_light_baker.cpp

```diff
@@ -1701,12 +1701,12 @@ Vector3 VoxelLightBaker::_compute_ray_trace_at_pos(const Vector3 &p_pos, const V
 			pos += advance;
 		}
 
-		if (cell != CHILD_EMPTY) {
+		if (unlikely(cell != CHILD_EMPTY)) {
 			for (int i = 0; i < 6; i++) {
 				//anisotropic read light
 				float amount = direction.dot(aniso_normal[i]);
 				if (amount < 0)
-					amount = 0;
+					continue;
```

Comment:
If guess it could even check `amount <= 0` to continue, but that probably doesn't matter much :D